### PR TITLE
Implement skill scale packet for client versions >= 20151223.

### PR DESCRIPTION
**Pull request by Asheraf**
[//]: # (**********************************)
[//]: # (** Fill in the following fields **)
[//]: # (**********************************)

[//]: # (Note: Lines beginning with syntax such as this one, are comments and will not be visible in your report!)

### Pull Request Prelude

[//]: # (Thank you for working on improving Hercules!)

[//]: # (Please complete these steps and check the following boxes by putting an `x` inside the brackets _before_ filing your Pull Request.)

- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR will be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed
this pr adds support for packet to show an area of effect for some skills using client lua files.

[//]: # (Describe at length, the changes that this pull request makes.)

**Affected Branches:** 
Master

[//]: # (Master? Slave?)

**Issues addressed:**
non

[//]: # (Issue Tracker Number if any.)

### Known Issues and TODO List

- [x] test on the given client date to verify if its correct : http://ro.gnjoy.com/news/notice/View.asp?BBSMode=10001&seq=6602&curpage=25

- [x] update skill_db based on skillinfolist.lua from kRO

[//]: # (Insert checklist here)
[//]: # (Syntax: - [ ] Checkbox)

[//]: # (**NOTE** Enable the setting "[√] Allow edits from maintainers." when creating your pull request if you have not already enabled it.)

[cont]: https://github.com/HerculesWS/Hercules/blob/master/CONTRIBUTING.md
[code]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style

### CODE REVIEW

**Comment by HerculesWSAPI** [comment]
This change is [Reviewable](https://reviewable.io/reviews/herculesws/hercules/1903)

File: src/map/clif.c [hunk 1]
```diff
@@ -19860,6 +19864,31 @@ void clif_parse_rodex_cancel_write_mail(int fd, struct map_session_data *sd)
 	rodex->clean(sd, 1);
 }
 
+void clif_skill_scale(struct block_list *bl, int src_id, int x, int y, uint16 skill_id, uint16 skill_lv, int casttime) {
```
Comment (4144) [inline src/map/clif.c]: { for functions should be on new line

Comment (Asheraf) [reply src/map/clif.c]: done

**Comment by germanafro** [comment]
since that commit our client regularly pops out this error when fighting mvps:
https://scontent.ftxl1-1.fna.fbcdn.net/v/t35.0-12/24139368_10155369927578640_34679060_o.jpg?oh=d7be329b33195bbcb5a4759b56de8b69&oe=5A1F2B5A

**Comment by Asheraf** [comment]
@germanafro Your lua files are outdated read this topic to get more information's http://herc.ws/board/topic/15498-how-to-fix-this/

**Pull request merged**